src/components/Overlay/DocumentOverlay.tsx

```diff
@@ -52,9 +52,9 @@ const getViewport = (containerDimensions: DOMRect): ViewportDimensions => {
   const width = 100
 
   // Hollow width ratio = 90% for smaller viewports
-  // and = 70% for bigger viewports
+  // and = 80% for bigger viewports
   const hollowWidthRatio =
-    containerDimensions.width < window.innerWidth ? 0.7 : 0.9
+    containerDimensions.width < window.innerWidth ? 0.8 : 0.9
```

Comment:
Oh does this mean the viewport shrinks when we're moving the `ContainerDimensionsProvider` deeper in the tree?

Reply:
I could see the `containerDimensions` were populated + what is expected but the resulting overlay sizing itself was smaller for the larger screens somehow.